### PR TITLE
Fixed a typo

### PR DESCRIPTION
**Pull request by SinithH**
## Description

Fixed a typo

### CODE REVIEW

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -337,7 +337,7 @@ The Gatsby `<Link />` component is for linking between pages within your site. F
 
 Gatsby.js is a _modern site generator_, which means there are no servers to setup or complicated databases to deploy. Instead, the Gatsby `build` command produces a directory of static HTML and JavaScript files which you can deploy to a static site hosting service.
 
-Try using [Surge](http://surge.sh/) for deploying your first Gatsby website. Surge is one of many "static site hosts" which make it possible to deploy Gatsby sites.
+Try using [Surge](http://surge.sh/) for deploying your first Gatsby website. Surge is one of many "static site hosts" which makes it possible to deploy Gatsby sites.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-one/index.md]: This was correct the way it was

File: docs/tutorial/part-five/index.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Alternatively, you can add fields by using the autocomplete shortcut (<kbd>Ctrl
 
 ![filesystem-autocomplete](filesystem-autocomplete.png)
 
-Try adding a number of fields to your query, pressing <kbd>Ctrl + Enter</kbd>
+Try adding a number of fields to your query, press <kbd>Ctrl + Enter</kbd>
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-five/index.md]: This is also fine the way it is

Comment (SinithH) [reply docs/tutorial/part-five/index.md]: Done !

File: docs/tutorial/part-one/index.md (hunk 1 above)
Comment (SinithH) [reply docs/tutorial/part-one/index.md]: Done !

File: docs/tutorial/part-five/index.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ You can access it when your site's development server is running—normally at
   <p>Your browser does not support the video element.</p>
 </video>
 
-Poke around the built-in `Site` "type" and see what fields are available on it -- including the `siteMetadata` object you queried earlier. Try opening GraphiQL and play with your data! Press <kbd>Ctrl + Space</kbd> (or use <kbd>Shift + Space</kbd> as an alternate keyboard shortcut) to bring up the autocomplete window and <kbd>Ctrl + Enter</kbd> to run the GraphQL query. You'll be using GraphiQL a lot more through the remainder of the tutorial.
+Poke around the built-in `Site` "type" and see what fields are available on it -- including the `siteMetadata` object you queried earlier. Try opening GraphiQL and play with your data! Pressing <kbd>Ctrl + Space</kbd> (or use <kbd>Shift + Space</kbd> as an alternate keyboard shortcut) to bring up the autocomplete window and <kbd>Ctrl + Enter</kbd> to run the GraphQL query. You'll be using GraphiQL a lot more through the remainder of the tutorial.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-five/index.md]: This was correct the way it was. Can you please revert this change?

File: docs/tutorial/part-five/index.md (hunk 2 above)
Comment (sidharthachatterjee) [reply docs/tutorial/part-five/index.md]: _pressing_ is correct here

**Comment by SinithH** [comment]
I made the changes you requsted and  opened a new pull request

File: docs/tutorial/part-five/index.md (hunk 2 above)
Comment (SinithH) [reply docs/tutorial/part-five/index.md]: Done

File: docs/tutorial/part-five/index.md (hunk 3 above)
Comment (SinithH) [reply docs/tutorial/part-five/index.md]: Done

File: docs/tutorial/part-five/index.md (hunk 2 above)
Comment (SinithH) [reply docs/tutorial/part-five/index.md]: Done

File: docs/tutorial/part-five/index.md (hunk 3 above)
Comment (vladar) [reply docs/tutorial/part-five/index.md]: It is still not reverted. It was `Press` and now it is `Pressing`

Comment (SinithH) [reply docs/tutorial/part-five/index.md]: Updated !
